api/DataCue.json

```diff
@@ -69,6 +69,38 @@
           }
         }
       },
+      "data": {
+        "__compat": {
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": "11"
+            },
+            "safari_ios": "mirror",
+            "samsunginternet_android": "mirror",
+            "webview_android": "mirror"
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
```

Comment:
It's not in https://wicg.github.io/datacue/#datacue-interface.

Reply:
Oops, closing.

Reply:
In a case like this when it's in the only implementation of the API I think it might still make sense to add it, just correctly marked as non-standard.

Reply:
That's a fair point -- for now though, I'd like to focus on adding standard track features, and then later look into the non-standard ones.  Standard features will be much easier to review!